Blockzilla/UIConstants.swift

```diff
@@ -90,5 +90,8 @@ struct UIConstants {
         static let toggleSafari = NSLocalizedString("Settings.toggleSafari", value: "Safari", comment: "Safari toggle label on settings screen")
         static let urlBarCancel = NSLocalizedString("URL.cancelLabel", value: "Cancel", comment: "Label for cancel button shown when entering a URL or search")
         static let urlTextPlaceholder = NSLocalizedString("URL.placeholderText", value: "Search or enter address", comment: "Placeholder text shown in the URL bar before the user navigates to a page")
+        static let toggleSectionSupport = NSLocalizedString("SUPPORT", comment: "Section label for support toggles")
+        static let labelSendAnonymousUsageData = NSLocalizedString("Send anonymous usage data", comment: "Label for toggle on main screen")
+        static let subtitleSendAnonymousUsageData = NSLocalizedString("More info…", comment: "Label for toggle on main screen")
```

Comment:
Please use `value:` for new strings (I've been trying to use `NSLocalizedString:value` for new strings like we do in Firefox). Hopefully I'll have a chance to also update the old strings so they're all using IDs before we do localization.